### PR TITLE
[expo] update definitions to expo-sdk@25.0

### PR DESCRIPTION
**Pull request by KonstantinKai**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://blog.expo.io/expo-sdk-v25-0-0-is-now-available-714d10a8c3f7
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/expo/index.d.ts [hunk 1]
```diff
@@ -632,7 +632,14 @@ export interface BarCodeScannerProps extends ViewProperties {
...
 }
 
-export class BarCodeScanner extends Component<BarCodeScannerProps> { }
+export class BarCodeScanner extends Component<BarCodeScannerProps> {
+    static Constants: {
+        TorchMode: {
+            on: string;
+            off: string
+        }
+    } & CameraConstants;
```
[the remark this answers is not in the document]
Comment (KonstantinKai) [reply types/expo/index.d.ts]: Yeah, sure.

**Comment by fhelwanger** [comment]
@KonstantinKai If I'm not mistaken, merging this would remove the definitions added by #23800 and #23801 from v24. Is this correct?

I'm using expo@24 and both changes apply to this version.

**Comment by KonstantinKai** [comment]
@fhelwanger #23800 will merge to v24, for v25 I'll insert changes manually.

File: types/expo/index.d.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (KonstantinKai) [reply types/expo/index.d.ts]: I've done :)

[the remark this answers is not in the document]
Comment (KonstantinKai) [reply types/expo/index.d.ts]: @fhelwanger I've done

**Comment by fhelwanger** [comment]
@KonstantinKai But isn't [this file](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23812/files#diff-f54e644ffd46ed0bd3692b7c20950cc3) overwriting what was done in #23800 and #23801?

Sorry if I'm missing something, it's my first time contributing to DefinitelyTyped 😄

**Comment by KonstantinKai** [comment]
@fhelwanger I moved this files with `git mv`, I don't think that there will be any problems :)

If you want, I can merge your changes to my fork.

**Comment by fhelwanger** [comment]
@KonstantinKai Ok, there's no need to merge 😉

**Comment by KonstantinKai** [comment]
@fhelwanger you're right this files will be conflicted :(

**Comment by andy-ms** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Pull request merged**